server/src/processor.rs

```diff
@@ -242,3 +226,13 @@ impl <'a> SignalProcessor<'a> {
         self.generate_response()
     }
 }
+
+fn is_higher(signal_1: &Signal, signal_2: &Signal) -> bool {
+    signal_1.height > signal_2.height
+        || (
+            signal_1.height == signal_2.height
+                        &&
+            signal_1.nonce > signal_2.nonce
+        )
+
+}
```

Comment:
Let's make this more descriptive

```
fn is_newer(v1: &Signal, v2: &Signal) -> bool {
    v1.height > v2.height || (v1.height == v2.height && v1.nonce > v2.nonce)
}

Reply:
i figured you'd hate the funky layout. youre worse than a HOA